reports/repo-active.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+#
+# Print repos:
+#   - having more than 1 contributor
```

Comment:
Why does an active repo have to have more than one contributor?

Reply:
It's doesn't, but are repositories with only 1 contributor interesting to report on? 🤔